WordPress/build.gradle

```diff
@@ -450,33 +449,33 @@ dependencies {
     implementation "org.greenrobot:eventbus-java:$eventBusVersion"
     implementation "com.squareup.retrofit2:retrofit:$squareupRetrofitVersion"
     implementation "org.apache.commons:commons-text:$apacheCommonsTextVersion"
-    implementation "com.airbnb.android:lottie:$lottieVersion"
+    implementation libs.lottie
     implementation "com.facebook.shimmer:shimmer:$facebookShimmerVersion"
     implementation ("com.github.yalantis:ucrop:$uCropVersion") {
         exclude group: 'androidx.core', module: 'core'
         exclude group: 'androidx.constraintlayout', module: 'constraintlayout'
         exclude group: 'androidx.appcompat', module: 'appcompat'
     }
-    implementation "com.github.bumptech.glide:glide:$glideVersion"
-    ksp "com.github.bumptech.glide:ksp:$glideVersion"
-    implementation "com.github.bumptech.glide:volley-integration:$glideVersion"
+    implementation libs.glide
+    ksp libs.glide.ksp
+    implementation libs.glide.volley.integration
     implementation "com.github.indexos.media-for-mobile:domain:$indexosMediaForMobileVersion"
     implementation "com.github.indexos.media-for-mobile:android:$indexosMediaForMobileVersion"
     implementation "com.zendesk:support:$zendeskVersion"
     implementation (name:'tenor-android-core-jetified', ext:'aar') // Jetified Tenor Gif library
-    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$kotlinxCoroutinesVersion"
-    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$kotlinxCoroutinesVersion"
+    implementation libs.kotlinx.coroutines.core
+    implementation libs.kotlinx.coroutines.android
     implementation "com.github.PhilJay:MPAndroidChart:$philjayMpAndroidChartVersion"
```

Comment:
I don't see any imports for this dependency! Seems like its no longer used. Can you verify it for safe removal? 

```
    implementation "com.github.indexos.media-for-mobile:domain:$indexosMediaForMobileVersion"
    implementation "com.github.indexos.media-for-mobile:android:$indexosMediaForMobileVersion"
    implementation "com.github.PhilJay:MPAndroidChart:$philjayMpAndroidChartVersion"
    implementation "org.jsoup:jsoup:$jsoupVersion"
```

Reply:
👋 @neeldoshii and thanks for taking a closer look into that! 🥇 

I recommend not removing any dependencies as part of your PR and instead create another issue describing the problem, for it to be then dealt with in isolation. This way this PR will also get reviewed faster.

FYI: Removing a dependency version is an easier problem than removing the dependency itself.

PS: Sometime a dependency might seems unused but is actually somehow used, on some configuration or a specific context. Thus, I cannot tell for sure if it is actually unused or not, not without going a bit deeper to first understand where that was introduced in the first place, how its usage was removed (in code) and why it wasn't removed as a dependency as well. I hope that all makes sense.